common/src/main/java/io/github/kurrycat/mpkmod/Main.java

```diff
@@ -126,6 +143,8 @@ public void loaded() {
         EventAPI.addListener(
                 new EventAPI.EventListener<OnRenderWorldOverlayEvent>(
                         e -> {
+                            if (!highlightLandingBlocks) return;
```

Comment:
This would work, but using the var as a default value for the visibility setting in the gui would probably be better, even if it would be more complicated